lib/lookup.js

```diff
@@ -96,6 +96,9 @@ function resolve(context, next) {
         context.options.script = rep.script;
       }
       context.module.flaky = context.options.failFlaky ? false : isMatch(rep.flaky);
+      context.module.verifyNodeGypCalled = rep['verify-node-gyp-called'] === true;
+      context.module.verifyNodeGypNotCalled = rep['verify-node-gyp-not-called'] === true;
```

Comment:
this could likely be `!!rep['verify-node-gyp-not-called']` or simply `rep['verify-node-gyp-not-called']`